internal/clients/team4/README.md

```diff
@@ -0,0 +1,18 @@
+# Team 4: Cherry Pitt 🍒
```

Comment:
Arents these files already in the main branch? I am confused as to what this diff actually shows now?

Reply:
Ahh now I see, it is not going onto the main branch..... Me understands